app/models/pot.rb

```diff
@@ -0,0 +1,6 @@
+class Pot < ApplicationRecord
+  belongs_to :owner, class_name: 'User', foreign_key: 'user_id'
+  has_many :gifts
+  has_many :users, through: :users_pot
+  validates :name, presence: true
+end
```

Comment:
I think we need to include dates here right? Maybe not start_date, but definitely close_date.